python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -397,6 +397,11 @@ def _annotate_image(fig, source, flags):
     if any(flags[flags_shape]):
         fig.text(0.87, 0.83, "SHAPE", color="red", fontweight="bold")
 
+    # rb score
+    fig.text(0.73, 0.79, f"RB:{source['spuriousness']:.03f}",
```

Comment:
It's worth wrapping this in an `if isfinite(source['spuriousness']):`, since these scores aren't included in our existing runs.